ClaudiaRojasSoto/Hello_Rails
# Feature: Initial Configuration with 'Hello World' in Rails

# Description:

## Summary:
This PR introduces a basic Rails application featuring a 'Hello World!' message on the main page.

### Details:
- [ ] Created a new Rails application named hellorails.
- [ ] Initialized the project with Git.
- [ ] Configured a Postgres database for the project.
- [ ] Generated a Pages controller with a hello action that renders a view.
- [ ] The view displays a 'Hello World' message within an `h1` element.
- [ ] Set the application's root route to direct to pages#hello.


### How to Test:
1. Clone the feature/initial-configuration branch.
2. Run bundle install to install dependencies.
3. Run rails db:create to create the database.
4. Run rails server to start the server.
5. Visit http://localhost:3000/ to see the 'Hello World' message.


### Relevant Files Created:
- [ ] File config/routes.rb: Root route configured to point to pages#hello.
- [ ] File app/controllers/pages_controller.rb: Added hello action.
- [ ] File app/views/pages/hello.html.erb: Added 'Hello World' message inside an `h1` element.

Please, review this feature and give some feedback to improve it. Thank you in advance.
Greetings, @ClaudiaRojasSoto
